app/src/display/status_screen.c

```diff
@@ -39,8 +44,17 @@ lv_obj_t *zmk_display_status_screen() {
                  0);
 #endif
 
+#if IS_ENABLED(CONFIG_ZMK_WIDGET_LAYER_STATUS)
+    zmk_widget_layer_status_init(&layer_status_widget, screen);
+    lv_obj_align(zmk_widget_layer_status_obj(&layer_status_widget), NULL, LV_ALIGN_IN_BOTTOM_LEFT,
+                 0, 0);
+
+    lv_label_set_text(zmk_version_label, "ZMK v0.1.0");
```

Comment:
At this point, we actually have some real info to show on displays, can we just ditch this made up version label altogether with this change?

Reply:
yes, will remove